Annex60/Fluid/Movers/BaseClasses/FlowControlled.mo

```diff
@@ -1,8 +1,31 @@
 within Annex60.Fluid.Movers.BaseClasses;
 partial model FlowControlled
   "Partial model for fan or pump with ideally controlled mass flow rate or head as input signal"
+  parameter Boolean use_record = false
```

Comment:
Why using underscore when all other parameters use camel case?

Also, don't use pump, use movers as it is also used for fans